examples/Crameri2012_case2/comparing plot_max_surface.ipynb

```diff
@@ -0,0 +1,215 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "9bccc364-6569-4908-ba59-85c8c1e8f538",
+   "metadata": {},
+   "source": [
+    "# Plot comparison\n",
+    "\n",
+    "Plot the comparison between Mandyoc results and UNDERWORLD, I2VEA and STAGE results."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "9d00eb6d-167a-4722-8b05-24d0458eeb0d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "import numpy.ma as ma\n",
+    "import matplotlib.pyplot as plt\n",
+    "import matplotlib.ticker as ticker\n",
+    "import os\n",
+    "import sys\n",
+    "import glob"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "7d7f9fca-bb0f-4c35-82db-aff094d20fb6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def load_and_plot(ax, file_name, color):\n",
+    "    \"\"\"\n",
+    "    Function to load and plot the UNDERWORLD, I2VEA and STAGE data\n",
+    "    \"\"\"\n",
+    "    times, surface_topo = np.loadtxt(\n",
+    "        \"{}.txt\".format(file_name), \n",
+    "        unpack=True, \n",
+    "        delimiter=\",\"\n",
+    "    )\n",
+    "    \n",
+    "    ax.plot(\n",
+    "        times,\n",
+    "        surface_topo, \n",
+    "        linestyle=\"-\",  \n",
+    "        label=file_name,\n",
+    "        color=color\n",
+    "    )"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "39e987e4-cd9a-4e45-aae6-9275ef6149f9",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "## Load Mandyoc surface data \n",
+    "Create a list with the name of the surface files generated by mandyoc:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "f22fe7e8-febe-48b3-bca2-1732274d8b24",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "surface_filenames = sorted(glob.glob(f\"sp_surface_global_*\"))\n",
+    "\n",
+    "nproc = len(glob.glob(\"step_0-rank_new*\"))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "b02a3a50-fcf8-45b0-90a1-1497b6f291a1",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "surface_topo_mandyoc = np.array([])\n",
+    "times = np.array([])\n",
+    "\n",
+    "for f in surface_filenames:\n",
+    "    step = int(f.split(\".\")[0].split(\"_\")[-1])\n",
+    "\n",
+    "    _s = np.loadtxt(f, skiprows=2, comments=\"P\")\n",
+    "    surface_topo_mandyoc = np.append(\n",
+    "        surface_topo_mandyoc, \n",
+    "        np.abs(np.max(_s))\n",
+    "    )\n",
+    "\n",
+    "    if step == 0 and int(surface_topo_mandyoc) == 0:\n",
+    "        surface_topo_mandyoc = 1.5e5\n",
+    "\n",
+    "    # read time\n",
+    "    with open(f\"time_{step}.txt\") as time_file:\n",
+    "        time_info = time_file.readline()\n",
+    "        _t = float(time_info.split(\":\")[-1])\n",
+    "        times = np.append(times, _t)\n",
+    "        \n",
```

Comment:
```suggestion
    "\n",
    "arg_sorted = np.argsort(times)\n",
    "times = times[arg_sorted]\n",
    "surface_topo_mandyoc = surface_topo_mandyoc[arg_sorted]\n",
    "\n",
```

surface_filenames are not being sorted correctly. This fix ordering to plot points in the correct order (time ordered).